src/csharp/Microsoft.Spark/Sql/DataFrameReader.cs

```diff
@@ -34,7 +35,25 @@ public DataFrameReader Format(string source)
             _jvmObject.Invoke("format", source);
             return this;
         }
-
+        
+        /// <summary>
+        /// Specifies the schema by using StructType object.
+        /// </summary>
+        /// <remarks>
+        /// Some data sources (e.g. JSON) can infer the input schema automatically
+        /// from data. By specifying the schema here, the underlying data source can
+        /// skip the schema inference step, and thus speed up data loading.
+        /// </remarks>
+        /// <param name="schema">class:`Microsoft.Spark.Sql.Types.StructType` object</param>
```

Comment:
```suggestion
        /// <param name="schema">The input schema</param>
```

Reply:
May I know why it's better to specify as `The input schema` than mention `StructType`?

Reply:
It's more descriptive than reiterating the type.

Reply:
Sounds good. Thanks for the suggestion.